### ISSUE TITLE
Handle OptionsInvalidNumberProvidedException 

### ISSUE THREAD

**Issue by obiefy** [akiyamaSM/larapoll#66]
When you using dashboard for creating new Poll, this exception will be thrown if you type the same text inside options field.
this is screenshot for form:
![Screenshot from 2019-04-16 15-42-27](https://user-images.githubusercontent.com/16496932/56214657-9553cc00-605e-11e9-8be1-99bbade03719.png)

and this for the exception
![Screenshot from 2019-04-16 15-44-29](https://user-images.githubusercontent.com/16496932/56214703-ad2b5000-605e-11e9-9b5e-dc2f7e4514cb.png)
.
the solution to handle this case is redirected to the create page with a warning alert to correct the option.

**Comment by obiefy** [comment]
if you type two options with the same text and another option with different text, this will make a poll with only two options. the first option for the duplicated text and the second for the different option.

**Comment by obiefy** [comment]
the bug is inside this function, it checks if the option text exists. 
```PHP
private function pushOption($option)
    {
        if(! in_array($option, $this->options_add)){
            $this->options_add[] = $option;
            return true;
        }
        return false;
    }
````

do we need to ignore this condition ??

**Comment by akiyamaSM** [comment]
@obiefy yes normally you can't have two options with the same value, it doesn't make any sense.
There must be a validation rule that detects duplication and it throws an exception

**Comment by obiefy** [comment]
Sorry, the exception comes from `generate()` function.

**Comment by akiyamaSM** [comment]
Thanks, I'll work on it tomorrow. unless if you want to propose a PR.

**Comment by obiefy** [comment]
I already fixed this issue, please review this PR #74

**Comment by akiyamaSM** [comment]
Thanks!
